python3.10/README.md

```diff
@@ -0,0 +1,136 @@
+Python 3.10.0 static compilation
```

Comment:
Use markup language to make article titles. you may use this tool https://dillinger.io/